app/views/games/show.html.erb

```diff
@@ -1,3 +1,29 @@
+<script>
+  $(function() {
+    $('.piece').draggable();
+    $('td').droppable({
+      drop: function(ev, ui) {
+        var valid_move = false;
+
+        $.ajax({
+          type: 'PUT',
+          async: false,
```

Comment:
@mscoutermarsh I put this here so that the valid_move variable would be updated before doing the if statement later. From the stuff I read though this seems like a bad thing since that kind of defeats the purpose of ajax. I had tried putting the `ui.draggable.draggable('option', 'revert', true);` stuff in a ajax.fail() function but it wasn't working that way. Do you have any suggestions on a better way to do this?